public/app/features/alerting/partials/notification_edit.html

```diff
@@ -18,6 +18,10 @@ <h3 class="page-sub-heading" ng-show="ctrl.isNew">New Notification Channel</h3>
           </select>
         </div>
       </div>
+      <div class="gf-form">
+        <a class="gf-form-label width-12" ng-click="ctrl.model.notifyOnce = !ctrl.model.notifyOnce;">{{ ctrl.model.notifyOnce ? 'Notify on state change' : 'Notify at most every' }}</a>
```

Comment:
I think this should be a checkbox. Clicking on the text to show other fields is not a UX pattern we use in other parts of Grafana.